src/frontend/Semantic_error.ml

```diff
@@ -152,6 +156,68 @@ module TypeError = struct
           (String.concat ~sep:"" lines)
           Fmt.(list UnsizedType.pp ~sep:comma)
           arg_tys
+    | IllTypedVariadicODE (name, arg_tys, args) ->
+        let types x = List.map ~f:(fun (_, t) -> t) x in
```

Comment:
small note: `(fun (_, t) -> t)` is also called `snd`

Reply:
Thanks. Fixed.